README.md

```diff
@@ -1,79 +1,81 @@
 # CloudShell Colony blueprint repository for Jenkins deployment
 
 This blueprint repository allows you to easily deploy Jenkins and connect it to your Cloudshell Colony account.
-After deployment this Jenkins instance could be immediately used to launch your pipelines against environments
-provided by Colony. It comes with a sample 'demo' pipeline, so that you will have an entrypoint from where
-you can start modeling your CI/CD flows using Jenkins and Colony  
+After deployment, this Jenkins instance can be immediately used to launch your pipelines against environments
+provided by Colony. It comes with a sample pipeline, so that you will have an entry point from which
+you can start modeling your CI/CD flows using Jenkins and Colony.
 
-### Repo consists of two blueprints:
+### This repo is comprised of two blueprints:
 
-**Jenkins**
+#### Jenkins
 
-This blueprint deploys a Sandbox with Jenkins itself. Flow is the following:
+This blueprint deploys a Sandbox with Jenkins itself. Its flow is comprises the following steps:
```

Comment:
his blueprint deploys a Sandbox using Jenkins. Its flow comprises the following steps: